src/cpp/session/modules/quarto/SessionQuarto.cpp

```diff
@@ -57,10 +57,12 @@ using namespace rstudio::core;
 
 const char * const kRStudioQuarto = "RSTUDIO_QUARTO";
 
-// ignored unused functions when quarto not enabled
-#ifndef QUARTO_ENABLED
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunused-function"
+#ifndef WIN32
+# define kQuartoCmd "quarto.cmd"
```

Comment:
Is this backwards? Shouldn't it be `quarto.cmd` and `quarto.exe` if we are building for Windows? Right now this is in a #ifndef WIN32.

Reply:
Doh. Will fix.